src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/Example4.xml

```diff
@@ -0,0 +1,13 @@
+/*
+FileTabCharacter
+eachLine = (default)false
+fileExtensions = java, xml
+
+
+*/
```

Comment:
While the html config "comment" is debatable (I am not a big fan), this is not valid xml. As we demand compilation for java files, we should have valid xml. Are we able to simply put this comment inside of a valid multiline xml comment? If fix is that easy, we should also update html file.

Reply:
Me too, but this is the only way for now. I tried putting the comment as such
```diff
+<!--
/*
FileTabCharacter
eachLine = (default)false
fileExtensions = java, xml


*/
+-->
```

but the `InlineConfigParser` is strict about it
> Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Config not specified on top.Please see other inputs for examples of what is required.


https://github.com/checkstyle/checkstyle/blob/79d3c5536760d2645b0676c5a7f31da3bb0eed72/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L164-L167

Maybe we can create an issue to extend and allow others?

Reply:
Please do, and take a few minutes to scope work required to allow us to parse config comment successfully with inner Java comment and without (only module config in xml comment).

Reply:
Opened at https://github.com/checkstyle/checkstyle/issues/13222

Reply:
I will remove the commit when #13223 gets merged. I continue with adding other tests.

Reply:
@nrmancuso I see that the PR/Issue might drag for some time. Let's leave it as is and we can come back to this issue at some point.

Reply:
Agreed.